src/ui/Modal.tsx

```diff
@@ -88,22 +92,12 @@ export const Modal = ({
       onOpenChange={onOpenChange}
     >
       <Dialog.Portal>
-        <Overlay onClick={onClose} />
```

Comment:
I like overly onclick close, why go bye bye?

Reply:
it still does.  just letting radix methods handle it via onOpenChange